### PR TITLE
seed ui feature

### PR DESCRIPTION
**Pull request by vigneshranganathan-eaton**
#### Changes proposed in this Pull Request:

- Included new feature Seed UI for react application

#### Screenshots / Screen Recording (if applicable)

-
image

#### To Test:

- Enter "npm run build:local"
- Select React in project framework
- Provide application name
- Select Seed UI in template
- Select Elsint true
- Select Prettier true
- npm start

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by joebochill** [comment]
@vigneshranganathan-eaton please fill out the PR template or Pull Requests will be summarily closed.

**Comment by joebochill** [comment]
Can you please update the README file to include information about the new template and add a Changelog entry for 3.1.0 (unreleased) that mentions the new template as well?

Since this is going to be (presumably) a react-only template, you'll want to add some conditional logic to only allow it if React is chosen as the framework.

File: src/constants/questions.ts [hunk 1]
```diff
@@ -31,4 +31,10 @@ export const QUESTIONS = {
         type: 'radio',
         choices: ['Blank', 'Basic Routing', 'Authentication'] as Template[],
     },
+    seedUItemplate: {
```
[the remark this answers is not in the document]
Comment (vigneshranganathan-eaton) [reply src/constants/questions.ts:34]: @joebochill 
Above template question is used for angular and react native project as well, If i add Seed UI option in above template, then i need to handle Seed UI option on Createangularproject and Createreactnativeproject function.

**Comment by vigneshranganathan-eaton** [comment]
> Can you please update the README file to include information about the new template and add a Changelog entry for 3.1.0 (unreleased) that mentions the new template as well?
> 
> Since this is going to be (presumably) a react-only template, you'll want to add some conditional logic to only allow it if React is chosen as the framework.



> 

@joebochill 
I have updated README file and changelog.
Seed UI option only available for react framework.
image

**Review by joebochill — APPROVED** [review]

**Pull request merged**